test/Tester/CancellationTests/GrainCancellationTokenTests.cs

```diff
@@ -22,7 +20,7 @@ protected override TestCluster CreateTestCluster()
             }
         }
 
-        [Theory, TestCategory("Functional"), TestCategory("BVT")]
+        [Theory, TestCategory("Functional"), TestCategory("BVT"), TestCategory("Cancellation")]
```

Comment:
Can you also reorder the categories so that BVT is the first one? Because Visual Studio 2015 (with latest update, as it wasn't the case before) UI will show the test in just 1 trait when you group them by trait, and it will pick the one that was first defined I believe.
And BVT is the category that we tend to run the most from VS while we are developing.

FYI, ordering makes no difference once we are running tests from the commandline (such as in CI), as if you decide to just run a particular trait, then it doesn't matter in which order it was defined for that test.